src/type/types.cpp

```diff
@@ -1275,4 +1275,9 @@ QuantifierType StringToQuantifierType(std::string str) {
   return QUANTIFIER_TYPE_NONE;
 }
 
+//===--------------------------------------------------------------------===//
+// Network Message types
+//===--------------------------------------------------------------------===//
+const std::string SqlStateErrorCodes::serialization_error =  "40001";
```

Comment:
Maybe we should make this an enum? There are going to be a lot more that we need to support:

https://www.postgresql.org/docs/9.6/static/errcodes-appendix.html